### PR TITLE
Added debian repository support

### PR DESCRIPTION
**Pull request by profgiuseppe**
Following instructions from https://github.com/natefoo/slurm-drmaa release page

### CODE REVIEW

File: galaxy.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+namespace: "progiuseppe"
+name: "galaxy-repos"
+version: "0.0.3"
+readme: "README.md"
+authors:
+    - "Helena Rasche (https://github.com/erasche)"
```
Comment (hexylena) [inline galaxy.yml]:
```suggestion
    - "Helena Rasche (https://github.com/hexylena)"
```

haven't used that username in about 2 years lol

Comment (profgiuseppe) [reply galaxy.yml]: I'm truly sorry Helena, copy-pasted it without double checking.

Comment (hexylena) [reply galaxy.yml]: Oh haha, no worries @profgiuseppe! no dramas, just had two people ping that handle recently and it's funny to me every time, I don't know where that is persisting that people see it. One of them mentioned it still shows up in gitter somehow, it's just amusing :)

**Comment by profgiuseppe** [comment]
This pull request may be too old, pool.sks-keyservers.net is no longer responding, Nate's key is not present in the ubuntu keyserver, it seems to be available in keys.openpgp.org. Fixed that.

**Comment by bvalot** [comment]
I have issue with keys.openpgp.org.
`sudo apt-key adv --keyserver hkp://keys.openpgp.org --recv-keys 18381AC8832160AF
Executing: /tmp/apt-key-gpghome.SxjKItWDGk/gpg.1.sh --keyserver hkp://keys.openpgp.org --recv-keys 18381AC8832160AF
gpg: key 52626447751B835F: new key but contains no user ID - skipped
gpg:       Quantité totale traitée : 1
gpg`

File: tasks/main.yml [hunk 2]
```diff
@@ -12,3 +12,17 @@
     baseurl: https://depot.galaxyproject.org/yum/el/$releasever/$basearch/
     gpgcheck: no
   when: ansible_os_family == "RedHat"
+
+- name: Add slurm-drmaa (Debian)
+  block:
+    - name: Ensure that source GPG key is known
+      ansible.builtin.apt_key:
+        keyserver: hkp://keys.openpgp.org
```
Comment (hexylena) [inline tasks/main.yml]:
```suggestion
        keyserver: hkp://keyserver.ubuntu.com
```

Comment (hexylena) [reply tasks/main.yml]: > Nate's key is not present in the ubuntu keyserver

ahhh jeez.

**Comment by hexylena** [comment]
A *lot* of the keyservers are dead at this point, unfortunately. Especially since ~2019 when an abuse was discovered that made keys unimportable, the network is dying. I'm not sure how to work around this

In the short term we can switch to another keyserver, <del>I think ubuntu works</del>, but, long term, I guess we have to wait for the community to decide.

File: tasks/main.yml (hunk 2 above)
Comment (natefoo) [reply tasks/main.yml]: It's here: http://keyserver.ubuntu.com/pks/lookup?search=18381AC8832160AF&fingerprint=on&op=index

Perhaps someone just submitted it?

Comment (bvalot) [reply tasks/main.yml:20]: Seems not valid as mentionned in my post:
`new key but contains no user ID - skipped`

**Comment by hexylena** [comment]
Ok, key is present now, so, we'll go with swapped out ubuntu keyservers.

**Comment by bvalot** [comment]
Do you need to change the **meta/main.yml** to add debian platform?

**Pull request merged**